requirements.txt

```diff
@@ -8,5 +8,6 @@ djangorestframework==3.12.4
 drf-nested-routers==0.93.3
 psycopg2==2.8.6
 python-decouple==3.4
+requests==2.20.0
 SQLAlchemy==1.4.18
-git+git://github.com/centerofci/sqlalchemy-filters@models_to_tables#egg=sqlalchemy_filters
\ No newline at end of file
+git+git://github.com/centerofci/sqlalchemy-filters@models_to_tables#egg=sqlalchemy_filters
```

Comment:
I wonder why that happened?

Reply:
i was wondering that too.

Reply:
I think a newline was added to the end of the file by my linter?